CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## [0.1.8] - 2024-05-20 12:00:00
+
+### Added
+
+- Updates the file calls in some of the calibration modules (`bequest_transmission.py`,  `deterministic_profiles.py`, `psid_data_setup.py`, and `transfer_distirbution.py`). These files called `.RData` files that were not included in the `ogusa` package, so users who pip installed ogusa could not instantiate the `Calibrate` class object. These changes allow for Jupyter notebook users to automatically download the data files from the master branch of the GitHub repository if they are not working in their own branch of the repository.
```

Comment:
@rickecon I think we should update this.  Perhaps:
```
Updates the `ogusa` package to include the zipped `psid_lifetime_income.csv.gz` file, which is now called in some calibration modules (`bequest_transmission.py`,  `deterministic_profiles.py`, and `transfer_distirbution.py`), but with an option for the user to provide their own custom datafile.  These changes allow for Jupyter notebook users to execute the `Calibration` class object and for those who install the `ogusa` package from PyPI to have the required datafile for the major calibration modules.
```

Reply:
@jdebacker. I just made this change.